pillar/vector/mitx.sls

```diff
@@ -530,3 +531,17 @@ vector:
         type: elasticsearch
         endpoint: 'http://operations-elasticsearch.query.consul:9200'
         index: logs-authlog-%Y.%W
+
+      s3_tracking:
+        inputs:
+          - tracking_log_timestamp_renamer
+        type: aws_s3
+        bucket: odl-residential-tracking-data
+        region: us-east-1
+        key_prefix: "%F-%H_"
+        encoding:
+          codec: ndjson
```

Comment:
Would it be possible to also Gzip the files that are uploaded?

Reply:
Yes, Vector Gzips them by default.